app/controllers/api/v1/users_controller.rb

```diff
@@ -1,61 +1,15 @@
 class Api::V1::UsersController < Api::V1::BaseController
-  # before_action :admin_access_required, only: [:index]
+  before_action :ensure_admin_user, only: [ :index ]
 
   def index
-    if current_user.admin?
-      users = []
-      User.all.order(:id).each{|u| users << sanitize_user_attributes(u)}
-
-      render json: {users: users}
-    else
-      render json: {errors: "access denied"}, status: 403
-    end
+      users = User.all.order(:id)
+      render json: users, serializer: UserSerializer
```

Comment:
I noticed that neither the `locations_controller.rb` nor the `donations_controller.rb` have actions that return a payload that is nested under a root JSON property like `{ users: ... }`. What's the format the client expects and do we need to adjust the other controllers to match this one or adjust this one to match the others?